build_and_push_images.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+IMAGES = [
+    dict(image_name='geometalab/osmaxx-mediator', dockerfile='Dockerfile.mediator'),
+    dict(image_name='geometalab/osmaxx-worker', dockerfile='Dockerfile.worker'),
+    dict(image_name='geometalab/osmaxx-frontend', dockerfile='Dockerfile.frontend'),
+]
+
+
+def docker_build(dockerfile, image_name, release, location='.'):
+    subprocess.check_call(['docker', 'build', '-f', dockerfile, '-t', '{}:{}'.format(image_name, release), location])
+
+
+def docker_push(release, image_name, *args, **kwargs):
+    subprocess.check_call(['docker', 'push', '{}:{}'.format(image_name, release)])
+
+
+if __name__ == '__main__':
+    import subprocess
+    release = subprocess.check_output(["git", "describe"]).strip().decode()
```

Comment:
Consider including the `--dirty` option, so that releases based on uncommitted changes can be identified. (Alternatively, check whether the working dir is clean and refuse operation is it isn't.)